alpha/notify-worker/README.md

```diff
@@ -0,0 +1,14 @@
+# NOTIFY-WORKER
+
+This is an example async worker implemented as a node process.
+
+This example uses several node packages not used elsewhere in this proof of concept:
+
+* [cron](https://github.com/ncb000gt/node-cron): Allows for cron syntax scheduling of logic within the node application
+* [nodemailer](https://github.com/nodemailer/nodemailer): Email module for node js, used in this instance to send a simple plaintext message to the SMTP debug container
+
+### SMTP debug server
```

Comment:
should move some of this into the main README as it is included in the docker-compose